src/appender/syslog-appender.lisp

```diff
@@ -7,7 +7,7 @@
 
 (in-package #:log4cl)
 
-(defclass syslog-appender (appender)
+(defclass syslog-appender (serialized-appender)
```

Comment:
I don't see how this can have any effect: `serialized-appender`: the `%lock` slot of the `serialized-appender` class is only used in the `stream-appender` sublcass. The syslog appender subclasses don't use the lock.